client/src/pages/Resource/index.js

```diff
@@ -0,0 +1,78 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+
+import { ResourceService, useService } from "../../services";
+
+import "./Resource.scss";
+
+export default function Resource() {
+	const [resource, setResource] = useState(undefined);
+	const resourceService = useService(ResourceService);
+	const { id } = useParams();
+
+	useEffect(() => {
+		resourceService.getOne(id).then(setResource);
+	}, [id, resourceService]);
+
+	if (!resource) {
+		return null;
+	}
+
+	return (
+		<>
+			<h2>
+				Resource &gt; <span>{resource.title}</span>
+			</h2>
+			<section>
+				<table aria-label="resource details" className="resource-details">
+					<tbody>
+						<tr>
+							<th>Title</th>
+							<td>{resource.title}</td>
+						</tr>
+						<tr>
+							<th>ID</th>
+							<td>{resource.id}</td>
+						</tr>
+						<tr>
+							<th>URL</th>
+							<td>
+								<a href={resource.url}>{formatUrl(resource.url)}</a>
+							</td>
+						</tr>
+						<tr>
+							<th>Topic</th>
+							<td>{resource.topic_name ?? <em>N/A</em>}</td>
+						</tr>
+						<tr>
+							<th>Suggested</th>
+							<td>
+								<time dateTime={resource.accession.toISOString()}>
```

Comment:
You _could_ extract the two uses of the `time` element to a separate component, but I think there'd still be some duplication. Either it's generic and you need to pass both the text to render _and_ the time:

```jsx
<Time
  dateTime={resource.accession}
  text={`${resource.accession.toLocaleString()} by ${resource.source_name}`}
/>
```

or it takes something like:

```jsx
<Time
  dateTime={resource.accession}
  name={resource.source_name}
/>
```

but then it can only be reused in a much smaller range of cases.